pandas/core/series.py

```diff
@@ -2266,36 +2266,61 @@ def _binop(self, other, func, level=None, fill_value=None):
 
     def combine(self, other, func, fill_value=None):
         """
+        Combine the Series with a Series or Scalar according to `func`.
+
         Perform elementwise binary operation on two Series using given function
-        with optional fill value when an index is missing from one Series or
-        the other
+        with optional `fill_value` when an index is missing from the Series or
+        the other value.
 
         Parameters
         ----------
-        other : Series or scalar value
-        func : function
-            Function that takes two scalars as inputs and return a scalar
-        fill_value : scalar value
+        other : Series or Scalar
+            The value(s) to be combined with the `Series`.
+        func : Function
+            `function` that takes two Scalars as inputs and returns a `bool`.
+        fill_value : Scalar
+            The optional value to assume when an index
+            is missing from one Series or the other,
             The default specifies to use the appropriate NaN value for
-            the underlying dtype of the Series
+            the underlying dtype of the Series.
```

Comment:
Can you finish the sentences close to 79 characters. I think this block fits in 3 lines.